src/resources/js/customizer-views-v2-controls.js

```diff
@@ -0,0 +1,240 @@
+			obj.controls.globalBackgroundColor
+		);
+	};
+
+	/**
+	 * Trigger control functions for the Events Bar section.
+	 *
+	 * @since TBD
+	 */
+	obj.handleEventsBar = function() {
+		// Only show the icon color control when the icon color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.eventsBarIconColorChoice,
+			obj.controls.eventsBarIconColor
+		);
+
+		// Only show the events bar background color control
+		// when the events bar background color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.eventsBarViewSelectorBackgroundColorChoice,
+			obj.controls.eventsBarViewSelectorBackgroundColor
+		);
+
+		// Only show the events bar view selector background color control
+		// when the events bar view selector background color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.eventsBarBackgroundColorChoice,
+			obj.controls.eventsBarBackgroundColor
+		);
+
+		// Only show the events bar border color control when the events bar border color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.eventsBarBorderColorChoice,
+			obj.controls.eventsBarBorderColor
+		);
+
+		// Only show the events bar button color control when the events bar button color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.eventsBarButtonColorChoice,
+			obj.controls.eventsBarButtonColor
+		);
+	};
+
+	/**
+	 * Trigger control functions for the Month View section.
+	 *
+	 * @since TBD
+	 */
+	obj.handleMonthView = function() {
+		// Only show the grid background color control when the grid background color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.monthGridBackgroundColorChoice,
+			obj.controls.monthGridBackgroundColor
+		);
+
+		// Only show the tooltip background color control when the grid background color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.monthGridBackgroundColorChoice,
+			obj.controls.monthTooltipBackgroundColor
+		);
+
+		// Only show the event span color control when the event span color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.monthMultidayEventBarColorChoice,
+			obj.controls.monthMultidayEventBarColor
+		);
+	};
+
+	/**
+	 * Trigger control functions for the Single Event section.
+	 *
+	 * @since TBD
+	 */
+	obj.handleSingleEvent = function() {
+		// Only show the single event title color control when the single event title color choice is set to custom.
+		obj.nestedColorControl(
+			obj.controls.singleEventTitleColorChoice,
+			obj.controls.singleEventTitleColor
+		);
+	};
+
+	/**
+	 * Trigger control functions for each section.
+	 *
+	 * @since TBD
+	 */
+	obj.init = function() {
+		obj.handleGlobalElements();
+
+		obj.handleEventsBar();
+
+		obj.handleMonthView();
+
+		obj.handleSingleEvent();
+	};
+
+	/**
+	 * Trigger our init function when customizer is ready.
+	 *
+	 * @since TBD
+	 */
+	wp.customize.bind( 'ready', function() {
+		obj.init();
+	} );
```

Comment:
I ❤️ the new implementation, this is so much cleaner! Just one suggestion.

```suggestion
	wp.customize.bind( 'ready', obj.init );
```